packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -709,12 +714,23 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
               asFeePayer
             );
           } else {
-            // else it means dapp passes legacy sdk v1 input data
-            // and the current connected wallet does not support it
-            throw new WalletNotSupportedMethod(
-              `Invalid transaction input data was provided, ${this.wallet?.name} expects AnyRawTransaction type. 
-              Please upgrade to the latest Aptos TS SDK https://github.com/aptos-labs/aptos-ts-sdk`
-            ).message;
+            const aptosConfig = new AptosConfig({
+              network: convertNetwork(this._network),
```

Comment:
```suggestion
              network: getAptosConfig(this._network),
```
see https://github.com/aptos-labs/aptos-wallet-adapter/pull/261